src/main/java/uk/gov/ons/ssdc/jobprocessor/schedule/RowChunkValidator.java

```diff
@@ -51,11 +51,17 @@ public void processChunk(Job job) {
       }
 
       for (ColumnValidator columnValidator : columnValidators) {
-        Optional<String> columnValidationErrors = columnValidator.validateRow(jobRow.getRowData());
+        try {
+          Optional<String> columnValidationErrors =
+              columnValidator.validateRow(jobRow.getRowData());
 
-        if (columnValidationErrors.isPresent()) {
+          if (columnValidationErrors.isPresent()) {
+            rowStatus = JobRowStatus.VALIDATED_ERROR;
+            rowValidationErrors.add(columnValidationErrors.get());
+          }
+        } catch (Exception ex) {
           rowStatus = JobRowStatus.VALIDATED_ERROR;
-          rowValidationErrors.add(columnValidationErrors.get());
+          rowValidationErrors.add(ex.getMessage());
```

Comment:
Given that these messages are intended to be viewed by external users, we might want to add a helpful prefix or something here. Maybe something like `"Unexpected technical failure, please report this to the dev team: <message>"` ?